apitools/base/protorpclite/messages.py

```diff
@@ -1321,7 +1321,9 @@ def validate_element(self, value):
         Raises:
           ValidationError if value is not expected type.
         """
-        if not isinstance(value, self.type):
+        if not (isinstance(value, self.type) or
+            (isinstance(value, (float, int)) and
```

Comment:
this test is looking a little hard to read -- can we just have

    if isinstance(value, int) and self.type == float:
      return

as the first few lines inside the `if`?

(note that i'm also trimming the logic to just what we need for now -- float -> int conversion is more worrisome, due to precision/rounding questions, i think?)